utils/common.js

```diff
@@ -0,0 +1,19 @@
+export function isEmpty(text) {
+	return !text;
+}
+export function isDuplicated(text){
+	var duplicated=false;
+	$('#tourneyPlayersList td').each(function(){
```

Comment:
This is not a generic utility method. This is hardly dependent on an HTML element. Instead, it should receive the list of players, not as DOM but as an array (model, no UI). Be consistent with what each module means.